tests/ssh/test_ssh_limit.py

```diff
@@ -42,7 +42,17 @@ def get_device_type(duthost):
 def modify_template(admin_session, template_path, additional_content, hwsku, type):
     admin_session.exec_command(TEMPLATE_BACKUP_COMMAND.format(template_path))
     admin_session.exec_command(TEMPLATE_CREATE_COMMAND.format(template_path))
-    admin_session.exec_command(LIMITS_CONF_TEMPLATE.format(hwsku, type, additional_content, template_path))
+
+    # run update template command and wait for template updated
+    config_file_content = []
+    while len(config_file_content) == 0:
+        stdin, stdout, stderr = admin_session.exec_command(LIMITS_CONF_TEMPLATE.format(hwsku, type, additional_content, template_path))
```

Comment:
This command sometimes will fail without error. can only randomly produce on testbed KVM.

Reply:
Retry few times, all test passed: 
ssh/test_ssh_limit.py::test_ssh_limits[] PASSED                          [100%]

Reply:
Could you explain in details "This command sometimes will fail without error" in PR description? If there is no error, why bothering fix.

Reply:
Fixed by remove the tee command.
The UT randomly failed because the template file not update, I found somestackover folow discussion about tee command will exist early when receive pipeline close signel.